engine/crates/federated-graph/src/render_sdl/render_federated_sdl.rs

```diff
@@ -28,62 +28,57 @@ pub fn render_federated_sdl(graph: &FederatedGraphV3) -> Result<String, fmt::Err
         sdl.push('\n');
     }
 
-    let query_type_exists = graph.objects.iter().any(|object| &graph[object.name] == "Query");
-    if !query_type_exists {
-        writeln!(sdl, "type Query\n")?;
-    }
-
     for (idx, object) in graph.objects.iter().enumerate() {
         let object_name = &graph[object.name];
         let is_query_root = graph.root_operation_types.query == ObjectId(idx);
 
-        if let Some(description) = object.description {
```

Comment:
The diff is not very readable from here, but it's just a bunch of code moving inside an if block.